### ISSUE TITLE
Domain name optionally as unicode after resolving

### ISSUE THREAD

**Issue by befranz** [imperviousinc/impervious-ios#3]
There's this lovely page https://🤝.🕰 - the first secure punycode domain site (by @pinheadmz) I know of. After resolving the name it only shows the URL as punycode. It would be cool to have the option to flip this back to unicode or at least to see unicode version somewhere.
![9128D567-E4EB-427C-AE0A-BE191CA53D0D_1_201_a](https://user-images.githubusercontent.com/46194732/143419586-cd98682c-662d-4f93-b0bf-f1b0e45a4b18.jpeg)

**Comment by buffrr** [comment]
Yeah emoji support would be cool! Some browsers don't support emojis because they consider it a security risk since many emojis look similar or don't look the same on different platforms. Safari does support them though we should look into the research they put into it to add this feature safely

I noticed Safari at least removes skin tone modifiers from emojis, but we have to do some more research

**Comment by befranz** [comment]
> I noticed Safari at least removes skin tone modifiers from emojis...

Does that mean all skintone variations of an emoji would resolve to just the original one?  Would be a bit of a drama for some people...

**Comment by buffrr** [comment]
> Does that mean all skintone variations of an emoji would resolve to just the original one? Would be a bit of a drama for some people...

Ha actually I just tried this and it seems like it doesn't do that for all emojis probably just ones that look too visually similar
